src/auth.py

```diff
@@ -48,6 +49,39 @@ def get_audiences():
     return getenv('BAYESIAN_JWT_AUDIENCE').split(',')
 
 
+def init_auth_sa_token():
+    """Initialize a service token from auth service."""
+    auth_server_url = getenv('AUTH_SERVICE_HOST', 'https://auth.prod-preview.openshift.io')
```

Comment:
We shouldn't expose the auth prod-preview url, instead return an empty string `''` and check for validity of `auth_url` before proceeding.